### ISSUE TITLE
Create queries to test performance of transition to TimescaleDB

### ISSUE THREAD

**Issue by aufdenkampe** [ODM2/ODM2DataSharingPortal#502]
As a stepping stone to **Replace InfluxDB with the TimescaleDB extension to PostgreSQL #434**, we need to start building some performance testing infrastructure.

Historically, the most challenging query for optimizing ODM2 performance has been the [`getSamplingFeatureDatasets()` call from the ODM2PythonAPI](https://github.com/ODM2/ODM2PythonAPI/blob/a5088ac8ecee9213e994fc0a71831825a9e05541/odm2api/services/readService.py#L969-L1043), with Sphinx documentation [here](http://odm2.github.io/ODM2PythonAPI/modules.html#module-odm2api.services.readService). However, the ODM2DataSharingPortal does not presently use this ODM2PythonAPI, which is based on SQLAlchemy. (Although we want to migrate toward that as part of #434). Nevertheless, this `getSamplingFeatureDatasets()` function might be a good reference.

For now, let's create a query which would be the functional equivalent of `getSamplingFeatureResults()`, bypassing the `Datasets` tables. Let's use the [ODM2OverviewSimplified](http://odm2.github.io/ODM2/schemas/ODM2_Current/diagrams/ODM2OverviewSimplified.html) ERD diagram as a guide.

In short, we want to:
-  start with a [`SamplingFeatureCode`](http://odm2.github.io/ODM2/schemas/ODM2_Current/columns/ODM2Core_SamplingFeatures_SamplingFeatureCode.html) (i.e a Site Code) from the [`ODM2Core.SamplingFeatures`](http://odm2.github.io/ODM2/schemas/ODM2_Current/tables/ODM2Core_SamplingFeatures.html) table;
- get all the associated [`ODM2Results.TimeSeriesResultValues`](http://odm2.github.io/ODM2/schemas/ODM2_Current/tables/ODM2Results_TimeSeriesResultValues.html) records.

Candidate sites are:
- WCC019
- RPB-2
- RPB-4

To highlight the benefits of TimescaleDB chunking, it might also be interesting to also add a test query that includes only fetching the last month of data.
